shared/set_configs/EMN.json

```diff
@@ -14,11 +15,87 @@
     ]
   },
   "SET_CORRECTIONS": [{
-    "match": {
-      "multiverseid": 414497
+      "match": {
+        "imageName": "brisela, voice of nightmares1"
+      },
+      "replace": {
+        "number": "15b",
+        "names": [
+          "Bruna, the Fading Light",
+          "Brisela, Voice of Nightmares"
```

Comment:
In trying to figure this out, we have the following to draw on:

# Two cards on one physical card

We have a great deal of precedent to draw on for how the front of a meld card should relate to the half of the back it represents.

## Split cards

One physical card and two `card` entries. Each `card` entry represents one Magic object and the entries are tied together by having `a` and `b` appended to their `number` and by having the other `card`'s name in their `name` field.

## Flip cards

Essentially the same as split cards just with top and bottom instead of left and right.

## Double-faced cards

Again, the same except front and back.

# One card on two physical cards

We have a lot less precedent to draw on for how the two back halves of a meld card should
relate to each other. And, unfortunately, the closest we have to precedent comes from an Un-set.

## B.F.M.

In this case each physical card gets its own `card` entry. Each `card` entry has a different `number` and they are related to each other by `imageName` and (somewhat poorly) the `names` fields

# So what do we do?

We essentially have two options, each with pros and cons, as well as details

## Three card entries

In this case each physical card gets a `card` entry for its front and shares a `card` entry for its back. There is one `card` entry for the melded back of the card.

### Problems to resolve

- Is `number` an array for the melded card or do we only have a `b` side for one of the halves.
- If only one `b` side, is the other half still an `a` side?
- If people are building software to track physical cards, how do they distinguish which half they have?

## Four card entries

In this case, each physical card gets a `card` entry for its front and for its back. There are two `card` entries for the melded back of the card.

### Problems to resolve

- Does a melded `card` entry get all three names or just the names of the `card` entries present on that single physical card?
- Does the front `card` entry get all three names or just the names of the `card` entries present on that single physical card?

# My position

Having laid all of that out, my position is that we should start with a four card entry solution. This is VERY important to me as I am primarily interested in tracking physical cards, so I would like to be able to refer to the top half of Brisela as distinct from the bottom half.

As for two names versus three, my biased, physical card centric, position makes me prefer the two name solutions in both cases.

Setting aside my biases, I see good arguments for putting all three names on the melded side and only two on the fronts. For one thing, without correction, that's what gatherer gives us for the fronts. Also, the other front isn't part of the front card in the way that it's part of the back.